NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/JavaScriptSnippet.cs

```diff
@@ -113,9 +113,9 @@ public string ScriptBody
                     }
                 }
 
+                Snippet = Snippet.Replace("instrumentationKey: \"INSTRUMENTATION_KEY\"",insertConfig );
                 // Return snippet
-                // Developer Note: If you recently updated the snippet and are now getting "FormatException: Input string was not in a correct format." you need to escape all the curly braces; '{' => '{{' and '}' => '}}'.
-                return string.Format(CultureInfo.InvariantCulture, Snippet, insertConfig, insertAuthUserContext);
+                return string.Format(CultureInfo.InvariantCulture, "{0}{1}" , Snippet , insertAuthUserContext);
```

Comment:
you don't need a `string.format` here. can use `string.concat` instead.

```suggestion
                return string.Concat(Snippet , insertAuthUserContext);
```